coding-standards/git.md

```diff
@@ -35,8 +36,10 @@ Commit Messages:
 -   Write clear, concise, and descriptive commit messages.
 -   Use the imperative mood ("add" instead of "added").
 -   Start with a capital letter.
--   Do not end the commit message with a period.
+-   Do not end the commit message with a period. 
 -   Example: `Add user authentication`
+-   Always put your commit message in the below context & frame a proper message like this, 
+> \*\*If applied this commit will, \<your commit message\>\*\* 
```

Comment:
![image](https://github.com/OsmosysSoftware/dev-standards/assets/16570290/722afcfa-5824-4e3e-9c89-faa7950398a4)
This does not look good. Can you add that example under the new point only and try to add some examples?

Reply:
Fixed format issues